Add `SEA Summer Suiji Showdown 4` wiki article (#15129)
ppy/osu-wiki: wiki/Tournaments/SSSS/4/en.md

| 69 | ::{ flag=VN }:: [PolarPowah](https://osu.ppy.sh/users/13229498) | 0.113 |
| 70 | ::{ flag=PH }:: [Yanoish](https://osu.ppy.sh/users/22712749) | 0.109 |
| 71 | ::{ flag=MY }:: [Ch3rryL](https://osu.ppy.sh/users/20272150) | 0.103 |
| 72 | ::{ flag=ID }:: [zreiF](https://osu.ppy.sh/users/16587039) | 0.097 |
| 73 | ::{ flag=MY }:: [WanHigh](https://osu.ppy.sh/users/24265798) | 0.060 |
| 74 | ::{ flag=VN }:: [JackGamerCYT](https://osu.ppy.sh/users/29909606) | 0.028 |

## Ruleset

### General rules

1. Match lobbies across the tournament will adhere to the following room settings:
   - Team Mode: `Team Vs`
   - Win Condition: [`ScoreV2`](/wiki/Gameplay/Score#scorev2)
2. The mappools for each round will be announced by the tournament management in advance before the actual matches take place.
3. Match schedules will be predetermined by the tournament management. If there are any teams who are unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the `#schedule` channel in the tournament's Discord server.
4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.

> 5. If a team does not able to show up with at least three players within **ten minutes** of the start time, their opponent gets to win by default.

Reviewer: ```suggestion
5. If a team does not show up with at least three players within **ten minutes** of the start time, their opponent gets to win by default.
```

6. If no staff or referee is available, the match will be postponed.

> 7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.

Reviewer: ```suggestion
7. **No Fail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```

8. If a player disconnects, it will be treated as if they had failed the beatmap.
   - A match can be rematched for disconnects that occur within a few seconds after the beatmap has been started by the referee.
9. Lag is not a valid reason to nullify a beatmap.
10. If any problems during the match occur, the tournament management will make a decision based on the referee's report.
11. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation.
    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player or their team may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the tournament management.

>     - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.

Reviewer: ```suggestion
    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be reported to the osu! team.
```
I think there's too much fluff otherwise

Reviewer: also "the team" [is vague](https://github.com/ppy/osu-wiki/issues/1698) -- maybe the support team instead?

### Tournament registration

1. Players are required to register into the tournament individually through [this form](https://forms.gle/qJqVwgBSkumg3zg9A).
   - In order to be eligible to play in the tournament, a player must:
     - Have the flag of any of the ASEAN member states (::{ flag=BN }:: Brunei Darussalam, ::{ flag=KH }:: Cambodia, ::{ flag=ID }:: Indonesia, ::{ flag=LA }:: Laos, ::{ flag=MY }:: Malaysia, ::{ flag=MM }:: Myanmar, ::{ flag=PH }:: The Philippines, ::{ flag=SG }:: Singapore, ::{ flag=TL }:: Timor-Leste, ::{ flag=TH }:: Thailand, or ::{ flag=VN }:: Vietnam) on their profile, ***or***
     - Be in possession of a valid national identity document (i.e. passport or ID card) issued by one of the aforementioned countries.
2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the tournament management.
3. The list of players who are deemed to be eligible to compete in the tournament will be published by the tournament management after the registration phase has ended.
4. Testplayers, referees, custom mappers, and mappool selectors may not participate as players in this tournament.

>    - Eliminated players are free to enlist as replay providers for the later stages of the tournament in accordance to the [official tournament support guidelines](/wiki/Tournaments/Official_support#staff). 

Reviewer: ```suggestion
   - Eliminated players are free to enlist as replay providers for the later stages of the tournament in accordance with the [official tournament support guidelines](/wiki/Tournaments/Official_support#staff). 
```

### Round-specific rules

#### Qualifier rules

> 1. The Qualifier will only be contested by players whose [BWS-adjusted ranking](/wiki/Tournaments/Badge-weighted_seeding) are outside of the Top 80 out of all players by the end of the registrations. **The top 80 players do not have to play in the Qualifier.**

Reviewer: ```suggestion
1. The Qualifier will only be contested by players whose [BWS-adjusted rankings](/wiki/Tournaments/Badge-weighted_seeding) are outside of the top 80 out of all players by the end of the registration. **The top 80 players do not have to play in the Qualifier.**
```

2. Players will have to sign up to one of the Qualifier lobbies that have been scheduled and prepared by the tournament management in advance.
3. In the lobby, players will have to consecutively play all of the Qualifier beatmaps in the order of NM1 -> NM2 -> NM3 -> NM4 -> NM5 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2 -> DT3.
4. Players **are not allowed** to ban any beatmaps in the Qualifiers.
5. Players **are not allowed** to join (or register for) more than one Qualifier lobby.
6. Based on their performance in the Qualifier, players will be ranked based on their **`%MAX` value**, which is the percentage of their score relative to the highest achieved score of all players in a map.
7. The 40 players with the **highest average `%MAX` value** across all the Qualifier beatmaps will advance to the team matchmaking phase.
   - If there are two (or more) players who share the same `%MAX` value, the player that holds the higher total raw score will be placed in the higher seed.

> 8. Failure to attend in any of the Qualifier lobbies will result in an outright elimination.

Reviewer: ```suggestion
8. Failure to attend in any of the Qualifier lobbies will result in immediate elimination.
```

#### Swiss round rules

1. The Swiss round will be held over the course of two weeks with a different mappool for each week.
2. Teams will be divided into two Swiss groups of ten teams each.
   - Group drawings will be done based on each team's average player ranks.
3. During each week, teams will play two matches against opponents in their group as determined by the [Swiss algorithm](https://help.start.gg/en/articles/2679435-start-gg-s-swiss-algorithm-and-additional-swiss-info).
4. Matches in the Swiss round will run until either team scores five points or total of eight beatmaps have been played (whichever comes first) with **no tiebreakers**.
   - When a match reaches the 4-4 score, it will be considered as a *draw*.
5. Based on their match results, teams will be awarded points that go to their Swiss round standings as follows:
   - Winning a match: +1 point
   - Drawing a match: +0.5 points
   - Losing a match: 0 points
6. The Swiss round standings are determined by (in order):
   - Points accumulated
   - Total number of beatmaps won
   - Beatmap difference (`# of beatmaps won` - `# of beatmaps lost`)
   - [Median Buchholz](https://en.wikipedia.org/wiki/Buchholz_system) score

>    - Head-to-head records between the tied teams   

Reviewer: ```suggestion
   - Head-to-head records between the tied teams
```

   - The result of an extra play-in match (if required)
7. Forfeiting a match will be treated as an outright loss with a -5 map difference to the forfeiting team.
8. The top 6 teams from each group based on the Swiss round standings will advance into the knock-out stage.
   - The top two teams will advance to the upper bracket directly, while the other four will advance to the lower bracket.

#### Knock-out stage rules

1. Teams will be matched against each other based on their Swiss round standings.
2. Teams will compete against each other using the double-elimination system.
3. The double-elimination system works as follows:
   - Teams that lose in the upper bracket can still play again in the lower bracket.
   - Teams that lose in the lower bracket will be eliminated from the tournament.

>    - In the Grand Final match, the winner of the the upper bracket will only need to win a single match in order to claim the championship title. The winner of the lower bracket, however, will need to win two matches and enforce a *bracket reset* in order to clinch the championship title.

Reviewer: ```suggestion
   - In the Grand Final match, the winner of the upper bracket will only need to win a single match in order to claim the championship title. The winner of the lower bracket, however, will need to win two matches and enforce a *bracket reset* in order to clinch the championship title.
```

### Match regulations

1. Prior to starting the match, team captains must run the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order. 
   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier lobbies.
2. Teams will have to ban **two beatmaps** from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match. 
   - Barring the tiebreaker, there are no restrictions as to which beatmaps may and may not be banned in a match.
   - Banning does not apply in the Qualifier lobbies.
3. **There will be no warm-up beatmaps to be played in the multiplayer lobby**. Players who are looking to warm up before the match are expected to do so by their own before the match commences.
4. Players are expected to exercise common sense in pick time windows.
   - If a player is unable to come up with a pick within a 90-second time window of their picking turn, the pick will be given to the other player.

> 5. In a FreeMod pick, teams have to play the beatmap with at least two unique mod combinations. Allowed mods are NoMod, Easy, HardRock, Hidden, Flashlight, or any possible combinations of the four.

Reviewer: ```suggestion
5. In a Free Mod pick, teams have to play the beatmap with at least two unique mod combinations. Allowed mods are No Mod, Easy, Hard Rock, Hidden, Flashlight, or any possible combinations of the four.
```
